Code/GraphMol/Descriptors/Lipinski.cpp

```diff
@@ -109,6 +110,70 @@ unsigned int calcNumRotatableBonds(const ROMol &mol, bool strict) {
     return m.get().countMatches(mol);
   }
 }
+#else
+const std::string NumRotatableBondsVersion = "1.0.3";
+unsigned int calcNumRotatableBonds(const ROMol &mol, bool useCactvsDefn) {
+  static ROMOL_SPTR origDefn_matcher;
```

Comment:
The problem is with lines 117 and 118.
I'm pretty sure this style of initialization (which used to be the way the RDKit did it everywhere) led to thread-related bugs on linux too.